src/components/home/partners/SecondaryPartners.jsx

```diff
@@ -0,0 +1,156 @@
+  return (
+    <Grid
+      container
+      sx={{
+        background: gradientStyle,
+        borderRadius: '30px',
+        minHeight: '500px',
+        marginBottom: '100px',
+        flexDirection: 'row'
+      }}
+      key={company}
+    >
+      <Grid item md={6} {...partnerGridStyle} order={logoOrder}>
+        <a href={website} target="_blank" rel="noopener noreferrer">
+          <Box
+            component={'img'}
+            alt={`${company} logo`}
+            aria-label={`${company} logo`}
+            src={partnerLogo}
+            mb={'150px'}
+            width={'250px'}
+          ></Box>
+        </a>
+      </Grid>
+      <Grid item pl={'6%'} {...partnerGridStyle} order={contentOrder}>
+        <Typography variant="body1" p={'5% 15% 3% 0'}>
+          {testimonial}
+        </Typography>
+        <Typography variant="caption" p={'15px 0 5% 0'}>
+          {testimonialAuthor}
+        </Typography>
+        {testimonialTwo ? (
+          <Typography variant="body1" display={'flex'} p={'40px 15% 15% 0'}>
+            {testimonialTwo}
+          </Typography>
+        ) : null}
+        {testimonialAuthorTwo ? (
+          <Typography variant="caption" pb={'5%'}>
+            {testimonialAuthorTwo}
+          </Typography>
+        ) : null}
+      </Grid>
+    </Grid>
+  );
+};
+
+const SmallScreenComponent = ({
+  index,
+  company,
+  testimonial,
+  testimonialTwo,
+  testimonialAuthorTwo,
+  testimonialAuthor,
+  partnerLogo,
+  website
+}) => {
+  return (
+    <Grid
+      container
+      sx={{
+        background:
+          'linear-gradient(180deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)',
+        borderRadius: '30px',
+        minHeight: 'auto',
+        marginBottom: '15%'
+      }}
+      key={company}
+    >
+      <Grid item {...partnerGridStyle}>
+        <a href={website} target="_blank" rel="noopener noreferrer">
+          <Box
+            component={'img'}
+            alt={`${company} logo`}
+            aria-label={`${company} logo`}
+            src={partnerLogo}
+            width={'100px'}
+          ></Box>
+        </a>
+      </Grid>
+      <Grid item {...partnerGridStyle}>
+        <Typography variant="body1" p={'5% 5% 5% 5%'}>
+          {testimonial}
+        </Typography>
+        <Typography variant="caption" p={'5% 5% 5% 5%'}>
+          {testimonialAuthor}
+        </Typography>
+        {testimonialTwo ? (
+          <Typography variant="body1" display={'flex'} p={'5% 5% 5% 5%'}>
+            {testimonialTwo}
+          </Typography>
+        ) : null}
+        {testimonialAuthorTwo ? (
+          <Typography variant="caption" p={'5% 5% 5% 5%'}>
+            {testimonialAuthorTwo}
+          </Typography>
+        ) : null}
+      </Grid>
+    </Grid>
+  );
+};
+
+const ResponsivePartnerDisplay = (props) => {
+  const theme = useTheme();
+  const isSmallAScreen = useMediaQuery(theme.breakpoints.down('md'));
+
+  if (isSmallAScreen) {
```

Comment:
1. why `isSmallAScreen` vs `isSmallScreen`?

2. would you be comfortable if line 140 - line 145 was a ternary instead, or do you like this better?

3. any reason `SmallScreenComponent` isn't self-closing, like `LargeScreenComponent`?

4. Do you feel all this is better than just saying:
```js
const SecondaryPartners = () => {
    isSmallAScreen ? (
        return (
            <Container>
                {secondaryPartnerList.map((partner, index) => (
                    <SmallScreenComponent />
                ))}
            </Container>
        )
    ) : (
        return (
            <Container>
                {secondaryPartnerList.map((partner, index) => (
                    <LargeScreenComponent />
                ))}
            </Container>
        )
}
```
which seemingly can be refactored down to just
`isSmallAScreen ? <SmallScreenComponent /> : <LargeScreenComponent />`

Reply:
1: late night typo, fixed in next push.
2: was switched to ternary, just didnt make it in last nights push.
3: auto tag closer did what it does and I didn't notice at the time but was fixed this morning for next push.

4:I changed it back into one function, seemed to be common practice to break the responsive display section out, that refactor you wrote would not work as is with changes needed it would be... 

```js
const SecondaryPartners = () => {
  const theme = useTheme();
  const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));

  return (
    <Container>
      {secondaryPartnerList.map((partner) =>
        isSmallScreen ? (
          <SmallScreenComponent key={partner.company} {...partner} />
        ) : (
          <LargeScreenComponent key={partner.company} {...partner} />
        )
      )}
    </Container>
  );
};
```
VS
```js
const ResponsivePartnerDisplay = (props) => {
  const theme = useTheme();
  const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));

  return isSmallScreen ? <SmallScreenComponent {...props} /> : <LargeScreenComponent {...props} />;
};

const SecondaryPartners = () => {
  return (
    <Container>
      {secondaryPartnerList.map((partner, index) => (
        <ResponsivePartnerDisplay key={partner.company} index={index} {...partner} />
      ))}
    </Container>
  );
};
```